data/xml/P14.xml

```diff
@@ -2633,7 +2654,8 @@
       <pages>706–711</pages>
       <url>P14-2115</url>
       <doi>10.3115/v1/P14-2115</doi>
-      <dataset>P14-2115.Datasets.tgz</dataset>
+      <attachment type="dataset">P14-2115.Datasets.tgz</attachment>
+      <attachment type="datasets">P14-2115.Datasets.tgz</attachment>
```

Comment:
Seems that this attachment type is added twice?

Reply:
Oh, drat. I checked for missing attachments before the tag conversion.